.github/workflows/lint.yaml

```diff
@@ -0,0 +1,34 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+
+name: lint
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: "[Stage] Dependencies"
```

Comment:
I tried removing this, and both the R CMD and lint actions failed because it couldn't install the Rmpi package, so I've added it back in. I think the `SystemRequirements` field might just be a free text field so it doesn't actually get installed (warning the installer they have to do something, but not doing it for them)?

Reply:
OK. Thanks for checking